### PR TITLE
Add README to tests for clarification

### PR DESCRIPTION
**Pull request by philoserf**
closes issue #42

### CODE REVIEW

**Comment by davidewatson** [comment]
I do not understand how this closes #42. @alika's comment was:

> @philoserf these curl test are for the CMA rest api. Notice:
>
> https://github.com/samsung-cnct/cluster-manager-api/blob/008a9bf4864822701703f2cd8144a0d60d5ccf4b/api/api.proto#L166
https://github.com/samsung-cnct/cma-vmware/blob/547f5bac57d14d71fb6d6cdb4e3b333846d9a074/api/api.proto#L116
I think you should either label them as such (move to tests\cma\...) or update them to use the CMA-vmware rest api.
 
Don't you need to either change these tests so that they are `cma-vmware` specific or move them to `cluster-manager-api`?

**Comment by philoserf** [comment]
I can be persuaded that it belongs outside of both.
I cannot see how something really intended to exercise this component via its parent belongs in the parent.

**Comment by davidewatson** [comment]
The problem is the JSON here will not work with the code here because `cma-vmware` and `cluster-manger-api` expose slightly different APIs. It is unfortunate, however, the tests belong with the code.

**Comment by philoserf** [comment]
Then I think the best choice is modify this to use cma-vmware API.

File: tests/create-cluster.sh [hunk 1]
```diff
@@ -20,43 +20,38 @@ DATA=$(
...
+        "password": "${NODE_PASSWORD}",
+        "labels": [
+          {
+            "name": "worker",
+            "value": "1"
+          }
+        ]
+      }
+    ],
+    "api_endpoint": "172.10.10.1:443"
```
Comment (davidewatson) [inline tests/create-cluster.sh]: Since endpoint is ignored I think we should leave this empty so as not to lead anyone astray.

Comment (philoserf) [reply tests/create-cluster.sh]: done

**Pull request merged**